### PR TITLE
Add compatibility with renovatebot

### PR DESCRIPTION
**Pull request by internalsystemerror**
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | no
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | yes

### Description

These changes will allow renovate to work as intended using the new shared configuration from https://github.com/laminas/.github/pull/19.

### CODE REVIEW

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ on:
     branches:
       - '[0-9]+.[0-9]+.x'
       - 'refs/pull/*'
+      - 'renovate/*'
```
Comment (internalsystemerror) [inline .github/workflows/continuous-integration.yml]: @Ocramius You mentioned removing the filter instead. Would I just make this an empty array to do that?

Comment (Ocramius) [reply .github/workflows/continuous-integration.yml]: Just removing these 3 lines suffices

Comment (internalsystemerror) [reply .github/workflows/continuous-integration.yml]: done.

**Review by Ocramius — APPROVED** [review]
💪 thanks @internalsystemerror!

**Pull request merged**